analyses/sample-distribution-analysis/02-multilayer-pie.R

```diff
@@ -0,0 +1,91 @@
+# This script creates a treemap and a multilayer pie chart to represent the
+# broad histologies, short histologies, and molecular subtypes within the dataset.
+#
+# This script uses the packages sunburstR, d3r, and treemap to produce the
+# visualizations.
+#
+# Chante Bethell for CCDL 2019
+
+# Load/install packages. This will be removed once a Dockerfile is established.
+source("00-install-packages.R", print.eval = FALSE)
+
+# Load source script
+source("01-filter-across-types.R")
```

Comment:
If I was looking at the scripts in this directory (and the README to a certain extent), I would expect them to run sequentially:
```sh
Rscript 00-install-packages.R
Rscript 01-filter-across-types.R
Rscript 02-multilayer-pie.R
```

If I did that and this is included here, I would run `01-filter-across-types.R` twice. I would remove this and the sourcing of the package installs from this script and add a shell script that runs each of them sequentially like what I have above. You can file a separate PR for the bash script as the very last pull request so all scripts are included.

Reply:
Also, because `01-filter-across-types.R` runs `source("00-install-package.R")` too, R wants me to restart to update a package that's already loaded. Not entirely sure what's going on there. It could have to do with the logic in `00-install-packages.R`.

Reply:
Ah, okay I think it _is_ the logic in `00-install-packages.R`. In a fresh session:

```R
> !require("ggplot2")
Loading required package: ggplot2
[1] FALSE
```

If you want to check if a package is installed without loading it in, you can use:
```R
!("ggplot2" %in% installed.packages())
```
And then you would load the package at the beginning of the script you needed it for _or_ stick to the `::` route (e.g., `ggplot2::`). This is preferable over what you have set up here because it gives you much more control over what is loaded.